app-arch/tar/patches/tar-1.29.patchset

```diff
@@ -5,26 +5,23 @@ Subject: gcc2 patch
 
 
 diff --git a/lib/wordsplit.h b/lib/wordsplit.h
-index d64cf2f..c93f53d 100644
+index e0d2559..acf1f56 100644
 --- a/lib/wordsplit.h
 +++ b/lib/wordsplit.h
-@@ -34,9 +34,9 @@ struct wordsplit
-   const char *ws_escape;
-   void (*ws_alloc_die) (struct wordsplit * wsp);
-   void (*ws_error) (const char *, ...)
--                   __attribute__ ((__format__ (__printf__, 1, 2)));
-+;
-   void (*ws_debug) (const char *, ...)
--                   __attribute__ ((__format__ (__printf__, 1, 2)));
-+;
+@@ -22,7 +22,7 @@
  
-   const char **ws_env;
-   const char *(*ws_getvar) (const char *, size_t, void *);
+ #include <stddef.h>
+ 
+-#if 2 < __GNUC__ + (7 <= __GNUC_MINOR__)
++#if 2 < __GNUC__ + (7 <= __GNUC_MINOR__) - __HAIKU__
```

Comment:
What isn't clear is that "__attribute__ ((__format__ spec))" is supposed to be available for GCC 2.95, so are you saying it isn't on Haiku?